pyproject.toml

```diff
@@ -0,0 +1,44 @@
+[build-system]
+requires = ["setuptools==67.2.0", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "cython_setuptools"
+description = "Cython setuptools integration"
+dynamic = ["version"]
+readme = "README.rst"
+requires-python = "~=3.7"
```

Comment:
3.10 ?

Reply:
The python version requirements depends of the compatibility of the code and actually it works starting from python3.7.